src/jit/jitconfigvalues.h

```diff
@@ -197,6 +210,7 @@ CONFIG_INTEGER(AltJitAssertOnNYI, W("AltJitAssertOnNYI"), 1) // Controls the Alt
 
 #if defined(_TARGET_X86_) || defined(_TARGET_AMD64_)
 CONFIG_INTEGER(EnableSSE3_4, W("EnableSSE3_4"), 1) // Enable SSE3, SSSE3, SSE 4.1 and 4.2 instruction set as default
+JIT_CONFIG_DWORD_INFO(EnableSSE3_4, W("EnableSSE3_4"), 1, "Enable SSE3, SSSE3, and SSE4 instruction set as default")
```

Comment:
We do not want to be adding new macros (that duplicate other macros) just to create the doc. The doc creation tool needs to be able to take the union of the knobs in both places.